### DISCUSSION TITLE
Update Tournament Client with twemoji Flags to match osu!web

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16418] [Ideas] [answered]
When creating bracket.json files for multiple taiko tournaments, I noticed the flags shown didn't match the flags seen on osu!web.
My best guess is lazer hasn't implemented twemoji flags whereas web has (osu-web#7875 / osu-wiki#6262)
![image](https://user-images.githubusercontent.com/74262407/149021216-5dbb26b1-c789-4352-8019-7fc998f8dcdc.png)

Edit: I realize this is similar to #15182 but I'm unsure how closely related the tournament client would be to the regular lazer client.

**Comment by bdach** [comment]
i have the twemoji assets generated but never got around to PRing them because flag sizings change and i couldn't be bothered to go and update them all. may revisit soon.

**Comment by frenzibyte** [comment, marked as answer]
Escalated to issue (#16422) to keep track of.
